charts/jaeger/README.md

```diff
@@ -1,440 +1,112 @@
-
-If you require a fully distributed architecture, you would need to manually create separate values files or fork this chart to create distinct Deployments for each role, utilizing the same Jaeger v2 binary but with different configurations (pipelines).
+This chart uses the **All-In-One** deployment model.
+- **Single Binary**: Runs as a `Deployment` scalable to multiple replicas.
+- **Stateless**: Can connect to external persistent storage (Elasticsearch, Cassandra) for production use.
+- **Default**: Memory storage (ephemeral), suitable for testing.
 
 ## Installing the Chart
 
 Add the Jaeger Tracing Helm repository:
-
 ```bash
 helm repo add jaegertracing https://jaegertracing.github.io/helm-charts
 ```
 
 To install a release named `jaeger`:
-
 ```bash
 helm install jaeger jaegertracing/jaeger
 ```
 
-By default, the chart deploys:
-
-- Jaeger All-in-One Deployment (combines collector, query, and internal components)
-- In-memory storage (non-persistent)
-
-### Overriding the Jaeger Version
-
-The chart version maps to a specific Jaeger version (appVersion). You can override this to use a different Jaeger image version without upgrading the chart:
+## Configuration
 
+### 1. In-Memory (Default)
+Ideal for testing. No persistence.
 ```bash
-helm install jaeger jaegertracing/jaeger \
-  --set allInOne.image.tag=2.13.0
+helm install jaeger jaegertracing/jaeger
 ```
 
-Or via a values file:
+### 2. Elasticsearch (Production Recommended)
+Configure Jaeger to connect to an existing Elasticsearch cluster using `extraEnv`.
 
+**values.yaml Example:**
 ```yaml
 allInOne:
-  image:
-    tag: "2.13.0"
+  extraEnv:
+    - name: SPAN_STORAGE_TYPE
+      value: elasticsearch
+    - name: ES_SERVER_URLS
+      value: http://elasticsearch:9200
+    - name: ES_USERNAME
+      value: elastic
+    - name: ES_PASSWORD
+      value: changeme
+```
+
+**Running Maintenance Jobs:**
+To run Index Cleaner or Rollover jobs, enable them and provide connection details:
+```yaml
+esIndexCleaner:
+  enabled: true
+  extraEnv:
+    - name: ES_SERVER_URLS
+      value: http://elasticsearch:9200
 ```
 
-## Configuration
-
-Jaeger v2 uses a YAML-based configuration format built on the OpenTelemetry Collector framework. The configuration is defined in the `config` section of the values file.
-
-See [Customizing the Chart Before Installing](https://helm.sh/docs/intro/using_helm/#customizing-the-chart-before-installing). To see all configurable options with detailed comments, visit the chart's [values.yaml](https://github.com/jaegertracing/helm-charts/blob/main/charts/jaeger/values.yaml), or run these configuration commands:
-
-```console
-$ helm show values jaegertracing/jaeger
+### 3. Cassandra
+To use Cassandra storage:
+```yaml
+storage:
+  type: cassandra
+  cassandra:
```

Comment:
this seems inconsistent with elasticsearch, should we not be using identical configuration mechanisms?